build/build-bundle.js

```diff
@@ -83,6 +88,7 @@ async function browserifyFile(entryPath, distPath) {
   }
 
   // Expose the audits, gatherers, and computed artifacts so they can be dynamically loaded.
+  // Exposed path relative to lighthouse-core/config/config-helpers.js (where loading occurs).
```

Comment:
important information, passed down over generations of build revisions, but only implicitly specified :)

Reply:
```suggestion
  // Exposed path MUST be a relative path from lighthouse-core/config/config-helpers.js (where loading occurs).
```

doesn't have to be spec style like this but I didn't totally understand this until rereading all the code to see what it did, any complete sentence might've helped :)